nilearn/image/__init__.py

```diff
@@ -4,7 +4,7 @@
 """
 from .resampling import resample_img, resample_to_img, reorder_img, \
     coord_transform
-from .image import high_variance_confounds, smooth_img, crop_img, \
+from .image import high_variance_confounds, smooth_img, crop_img, bin_img, \
```

Comment:
```suggestion
from .image import high_variance_confounds, smooth_img, crop_img, binarize_img, \
```

Reply:
I think that there is little cost in using the full name

Reply:
fixed